spec/parser_spec.rb

```diff
@@ -15,6 +19,177 @@
       it { is_expected.to be_a Result }
     end
 
+    context "with a minimal valid result" do
+      let(:fixture) { :minimal_valid }
+
+      describe "#posts[0]" do
+        let(:post) { result.posts[0] }
```

Comment:
There's no functional difference (that I know of), but I'm thinking maybe this could be:

```Ruby
subject(:post) { result.posts[0] }
```

Since it would indicate the it's the post that's being tested.